pkg/restore/client.go

```diff
@@ -814,6 +814,7 @@ func (rc *Client) GoValidateChecksum(
 			}
 			elapsed := time.Since(start)
 			summary.CollectDuration("restore checksum", elapsed)
+			summary.CollectSuccessUnit("table checksumed", 1, elapsed)
```

Comment:
Typo?
```suggestion
			summary.CollectSuccessUnit("table checksum", 1, elapsed)
```

Reply:
`CollectSuccessUnit` seems to collect the 'unit' of some entity. So "table checksummed" means "one table that has been restore and checksummed". 

Seems this would be confusing, let's change it. 

(BTW, this name currently DOESN'T appear in any log.😅)